cmd/bblfsh-sdk-tools/cmd/fixtures.go

```diff
@@ -62,72 +71,89 @@ func (c *FixturesCommand) generateFixtures(filename string) error {
 		return err
 	}
 
-	native, err := c.getNative(source, filename)
+	//UASTv2 sematic, native
+	err = c.writeUASTv2(source, filename, c.ExtNative, protocol2.Mode_Native)
 	if err != nil {
 		return err
 	}
 
-	err = c.writeResult(filename, native, c.ExtNative)
+	err = c.writeUASTv2(source, filename, c.ExtUast, protocol2.Mode_Semantic)
 	if err != nil {
 		return err
 	}
 
-	uast, err := c.getUast(source, filename)
+	//UASTv1 legacy, proto
+	resp, err := c.writeLegacyUASTv1(source, filename)
 	if err != nil {
 		return err
 	}
 
-	err = c.writeResult(filename, uast.String(), c.ExtUast)
+	if c.ExtProto != "" {
+		return c.writeProto(resp, filename, c.ExtProto)
```

Comment:
I'm not sure why we need to write legacy protos, but if so, it also makes sense to write v2 binary format as well. The proposed extensions is `.g`.

Reply:
Idea was just to be consistent with test output.